### PR TITLE
Feature event logging

### PR DESCRIPTION
**Pull request by zmelnick**
This is an open pull request for Event Logging. As tickets are closed and parts of Event Logging are implemented, this pull request will update. All people adding issues to this request please comment clearly on what you are doing,

### CODE REVIEW

File: Product/Production/Adapters/AdminDistribution_a0/src/main/webapp/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -0,0 +1,27 @@
```
Comment (bhumphrey) [inline Product/Production/Adapters/AdminDistribution_a0/src/main/webapp/META-INF/MANIFEST.MF:0]: shouldn't be committed

File: Product/Production/Adapters/DocumentQuery_a0/src/main/webapp/META-INF/MANIFEST.MF [hunk 2]
```diff
@@ -0,0 +1,30 @@
+Manifest-Version: 1.0
```
Comment (bhumphrey) [inline Product/Production/Adapters/DocumentQuery_a0/src/main/webapp/META-INF/MANIFEST.MF:1]: this is generated by the build. this shouldn't be committed.

File: Product/Production/Adapters/DocumentRetrieve_a0/src/main/webapp/META-INF/MANIFEST.MF [hunk 3]
```diff
@@ -0,0 +1,30 @@
+Manifest-Version: 1.0
```
Comment (bhumphrey) [inline Product/Production/Adapters/DocumentRetrieve_a0/src/main/webapp/META-INF/MANIFEST.MF:1]: same.

File: Product/Production/Adapters/DocumentSubmission_a0/src/main/webapp/META-INF/MANIFEST.MF [hunk 4]
```diff
@@ -0,0 +1,27 @@
+Manifest-Version: 1.0
```
Comment (bhumphrey) [inline Product/Production/Adapters/DocumentSubmission_a0/src/main/webapp/META-INF/MANIFEST.MF:1]: same

File: Product/Production/Adapters/General/CONNECTAdapterWeb/src/main/webapp/META-INF/MANIFEST.MF [hunk 5]
```diff
@@ -0,0 +1,32 @@
+Manifest-Version: 1.0
```
Comment (bhumphrey) [inline Product/Production/Adapters/General/CONNECTAdapterWeb/src/main/webapp/META-INF/MANIFEST.MF:1]: same

File: Product/Production/Adapters/General/CONNECTConsumerPreferencesProfileGUI/src/main/webapp/META-INF/MANIFEST.MF [hunk 6]
```diff
@@ -0,0 +1,44 @@
+Manifest-Version: 1.0
```
Comment (bhumphrey) [inline Product/Production/Adapters/General/CONNECTConsumerPreferencesProfileGUI/src/main/webapp/META-INF/MANIFEST.MF:1]: same

File: Product/Production/Adapters/General/CONNECTDeferredQueueManagerGUI/src/main/webapp/META-INF/MANIFEST.MF [hunk 7]
```diff
@@ -0,0 +1,43 @@
+Manifest-Version: 1.0
```
Comment (bhumphrey) [inline Product/Production/Adapters/General/CONNECTDeferredQueueManagerGUI/src/main/webapp/META-INF/MANIFEST.MF:1]: same

File: Product/Production/Adapters/General/CONNECTUniversalClientGUI/src/main/webapp/META-INF/MANIFEST.MF [hunk 8]
```diff
@@ -0,0 +1,47 @@
+Manifest-Version: 1.0
```
Comment (bhumphrey) [inline Product/Production/Adapters/General/CONNECTUniversalClientGUI/src/main/webapp/META-INF/MANIFEST.MF:1]: same

File: Product/Production/Adapters/PatientDiscovery_a0/src/main/webapp/META-INF/MANIFEST.MF [hunk 9]
```diff
@@ -0,0 +1,28 @@
+Manifest-Version: 1.0
```
Comment (bhumphrey) [inline Product/Production/Adapters/PatientDiscovery_a0/src/main/webapp/META-INF/MANIFEST.MF:1]: same

File: Product/Production/Gateway/DocumentSubmission_20/src/main/webapp/WEB-INF/cxf-servlet.xml [hunk 10]
```diff
@@ -38,25 +46,39 @@ http://cxf.apache.org/jaxws http://cxf.apache.org/schemas/jaxws.xsd">
...
         <jaxws:handlers>
             <ref bean="SOAPHeaderHandler" />
-            <ref bean="TransactionHandler"/>
+            <ref bean="TransactionHandler" />
         </jaxws:handlers>
     </jaxws:endpoint>
 
     <!-- Entity services -->
+
+    <bean id="entityDocSubmissionOrchImpl" class="gov.hhs.fha.nhinc.docsubmission.entity.EntityDocSubmissionOrchImpl" />
```
Comment (rhalfertycgi) [inline Product/Production/Gateway/DocumentSubmission_20/src/main/webapp/WEB-INF/cxf-servlet.xml:55]: The g0 and g1 services seem to be implemented differently.

**Comment by bhumphrey** [comment]
closing to merge to feature-EventLogging

**Pull request closed without merging**